fe/fe-core/src/main/java/com/starrocks/http/HttpServer.java

```diff
@@ -221,7 +222,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
                     .addLast(new ChunkedWriteHandler())
                     // add content compressor
                     .addLast(new CustomHttpContentCompressor())
-                    .addLast(new HttpServerHandler(controller));
+                    .addLast(new DefaultEventLoopGroup(), new HttpServerHandler(controller));
```

Comment:
How many threads in DefaultEventLoopGroup?  we should configure a proper number of threads.